app/feeswapper/management/commands/feeSwapper.py

```diff
@@ -0,0 +1,185 @@
+
+# Get all ERC-20 tokens and associated Uniswap Exchange addresses (if any).
+# Returns a dict with keys being token ERC-20 contracts and associated name/symbol/Uniswap exchange address/wallet balance in that token
+class Command(BaseCommand):
+        """ Check FEE_ADDRESS for ERC-20 token balances and tries to convert to ETH using Uniswap if exchange is available"""
+
+        help="Convert ERC-20 token balances in .env/FEE_ADDRESS to ETH where a Uniswap exchange is available"
+
+        network = ''   
+        web3 = ''
+        factoryContract = ''
+        tokenList = ''
+        sg = sendgrid.SendGridAPIClient(apikey=settings.SENDGRID_API_KEY)
+        
+        def getTokenList(self):
+                """ Queries the blockscout API for ERC-20 token balances
+                
+                Returns:
+                  tokenList: A dict with the key being the ERC-20 token address and the value being an object containing
+                  the token name, token symbol, Uniswap exchange address, and the token balance held by the FEE_ADDRESS
+                """
+                
+                tokenList = {}
+                r = requests.get('https://blockscout.com/eth/'+self.network+'/api?module=account&action=tokenlist&address='+settings.FEE_ADDRESS)
+                print(r.json())
+
+                # Check if API is functioning or not and return null token list if so
+                if not r.ok:
```

Comment:
Lets combine these checks that return an empty dictionary

Reply:
Done locally.  Will include in next commit.